src/class-methods/syntax/class-decl.template

```diff
@@ -0,0 +1,13 @@
+// Copyright (C) 2018 Bloomberg LP. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
```

Comment:
no need to create these template files if we reuse the templates from valid and invalid.